scripts/react-next.diff

```diff
@@ -156,6 +146,35 @@ index ef6533f4b..bd5633dc4 100644
        });
      });
    });
+diff --git a/packages/material-ui/src/Unstable_TrapFocus/Unstable_TrapFocus.js b/packages/material-ui/src/Unstable_TrapFocus/Unstable_TrapFocus.js
+index 97fac726d..e43c9f5d4 100644
+--- a/packages/material-ui/src/Unstable_TrapFocus/Unstable_TrapFocus.js
++++ b/packages/material-ui/src/Unstable_TrapFocus/Unstable_TrapFocus.js
```

Comment:
Can't come up with a reliable isolated test for this change so this stays strictly in this patch for now. There's definitely an issue with testing-library (we probably need to use a fork of `cleanup` because the new one is not explained and targetted at jest) and likely an issue with JSDOM.

Reply:
Also very likely that there's an issue on our end. In the end we're reaching into a lot of internals.